client/asyncns.c

```diff
@@ -77,7 +77,7 @@ enum {
 };
 
 struct asyncns {
-    int fds[MESSAGE_FD_MAX];
+    int fds[4];
```

Comment:
Oh, this is 3rd party code? Why it is not in some `vendor` subdirectory or something. :-)